recipes/libzip/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "libzip" %}
+{% set version = "1.4.0" %}
+{% set sha256 = "759a89690e155ca52247638b9f97e16e48fbb6077abb7ce2d921dd5b81259940" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://libzip.org/download/libzip-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # This disables the "malloc" test which also doesn't compile on Windows.
+    # This patch is taken from a commit post 1.4.0 release
+    - 0001-Unhook-malloc.c-from-build.patch  # [win]
+    # This disables a test that the library developers had disabled at one point
+    # https://github.com/nih-at/libzip/commit/57b97e9f069f8f9164ec6877bc00edb00cdb85b3
+    # Disabling for our build for now, pending fix or removal in library
+    - 0002-disable-nonrandomopentest-from-build.patch  # [win]
+    - 0003-Use-cmake-E-tar-to-extract-test-data.patch  # [win]
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - cmake
+    - zlib
+    - bzip2
+    - python  # [win]
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - vc 14  # [win and py36]
+    # perl is needed to run the package tests
+    - perl
```

Comment:
This is going to cause some problems while we're still on conda-build 2, because it does some automatic pinning stuff for perl. If you make it `perl >0`, it'll work correctly (https://github.com/conda-forge/parallel-feedstock/pull/1#issuecomment-362510209).